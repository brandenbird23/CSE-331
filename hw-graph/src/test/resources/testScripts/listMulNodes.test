# listMulNodes test file
# tests if you can put multiple nodes in one graph

# create a graph
CreateGraph graph1

# create nodes
AddNode graph1 node1
AddNode graph1 node2
AddNode graph1 node3

# Print the nodes in the graph
ListNodes graph1