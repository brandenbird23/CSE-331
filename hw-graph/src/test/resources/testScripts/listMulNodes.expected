# listMulNodes test file
# tests if you can put multiple nodes in one graph

# create a graph
created graph graph1

# create nodes
added node node1 to graph1
added node node2 to graph1
added node node3 to graph1

# Print the nodes in the graph
graph1 contains: node1 node2 node3